apps/examples/testcase/le_tc/network/tc_net_igmp.c

```diff
@@ -170,8 +161,8 @@ static void tc_etharp_free_entry(void)
 */
 static void tc_free_etharp_q(void)
 {
-	struct etharp_q_entry *q;
-	q = (struct etharp_q_entry *)malloc(sizeof (struct etharp_q_entry));
+	struct etharp_q_entry *q = NULL;
+	q = (struct etharp_q_entry *)malloc(sizeof(struct etharp_q_entry));
 
 	free_etharp_q(q);
```

Comment:
I left comments before. This function is internal fucntion on lwip/netif. How do you call it?